src/main/java/com/endava/soa_5/basic/HomeWork.java

```diff
@@ -0,0 +1,27 @@
+package com.endava.soa_5.basic;
+
+public class HomeWork {
+
+    public static void main(String[] args) {
+
+        Exercise_1 exercise_1 = new Exercise_1();
+        exercise_1.setTotalSec(5000);
+        System.out.println(exercise_1.messageForResult());
+
+        Exercise_2 exercise_2 = new Exercise_2();
+        System.out.println(exercise_2.setNumberForResult(8));
+
+        Exercise_3 exercise_3 = new Exercise_3();
+        System.out.println(exercise_3.setDayForResult(9));
+
+        Exercise_4 exercise_4 = new Exercise_4();
+        System.out.println(exercise_4.sumForLoopResult(7));
+
+        Exercise_4 exercise_4_1 = new Exercise_4();
+        System.out.println(exercise_4_1.sumForLoopResult(8));
+
+        Exercise_4 exercise_4_2 = new Exercise_4();
+        System.out.println(exercise_4_2.sumForLoopResult(9));
```

Comment:
Why use only sumForLoopResult? Please use sumWhileResult and sumDoWhileResult as well

Reply:
Done!